src/guacd/conf-file.c

```diff
@@ -203,6 +203,7 @@ guacd_config* guacd_conf_load() {
     /* Notify of errors preventing reading */
     else if (errno != ENOENT) {
         fprintf(stderr, "Unable to open \"" GUACD_CONF_FILE "\": %s\n", strerror(errno));
+	free(conf);
```

Comment:
Please check your style - this indentation should match the indentation around it.  See the following document:
http://guacamole.incubator.apache.org/guac-style/